ceph-dev-pipeline/build/Jenkinsfile

```diff
@@ -0,0 +1,229 @@
+ceph_build_repo = "https://github.com/ceph/ceph-build"
+ceph_build_branch = "jenkinsfile"
+base_node_label = "gigantic"
+ubuntu_releases = [
+  "noble", // 24.04
+  "jammy", // 22.04
+  "focal", // 20.04
+]
+debian_releases = [
+  "bookworm", // v12 (2023)
+  "bullseye", // v11 (2021)
+]
```

Comment:
Embarrassing question... Why is there no RPM distro list?

Reply:
Since those distros don't use codenames, we can use a regex like `^(centos|rhel|fedora).*` - we don't need lists.